src/pages/OpenOrdersPage.tsx

```diff
@@ -6,10 +6,13 @@ import {
   useMarketInfos,
 } from '../utils/markets';
 import OpenOrderTable from '../components/UserInfoTable/OpenOrderTable';
-import { Button } from 'antd';
+import { Button, Row } from 'antd';
 import { OrderWithMarketAndMarketName } from '../utils/types';
+import { useWallet } from '../utils/wallet';
+import WalletConnect from '../components/WalletConnect';
 
 export default function OpenOrdersPage() {
+  const { connected } = useWallet(0);
```

Comment:
`useWallet` doesn't take any arguments as it is currently written. Is this a typo maybe?

Reply:
Yep typo! Thanks for the catch